_people/RalphEtienneCummings.md

```diff
@@ -0,0 +1,12 @@
+---
+first_name: Ralph
+last_name: Etienne-Cummings
+title: Professor of Electrical & Computer Engineering
+school: Johns Hopkins University
+website: https://engineering.jhu.edu/csms/
```

Comment:
Can this link to a page about him individually, since we have the lab link below